src/app/components/navigation/nav-entry/nav-entry.component.html

```diff
@@ -6,23 +6,21 @@
 
 <ng-container *ngIf="entry.children">
   <ng-container *ngIf="hasPermission(entry) | async">
-    <mat-list-item
-      [routerLink]="entry.route"
-      routerLinkActive="active-parent-link"
-      (click)="toggleChildren(entry)"
-      #isRouteActive="routerLinkActive"
-    >
-      <span matListItemAvatar>
-        <mat-icon class="list-item-avatar" [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}</mat-icon>
-      </span>
-      <a matListItemTitle>
-        {{ entry.i18n | translate }}
-        <span>
+    <a mad-sidebar-item [routerLink]="entry.route"
+       routerLinkActive="active-sidebar-item"
+       (click)="toggleChildren(entry)"
+       #isRouteActive="routerLinkActive">
+      <div class="content-display-flex">
+        <div class="display-inherit-gap">
+          <mat-icon [ngStyle]="{ 'margin-left': sublevel * 8 + 'px' }">{{ entry.matIcon }}</mat-icon>
```

Comment:
Seems like the same thing as above :)

Reply:
Fixed. Created a service with calculation style object